docs/articles/blog/2019-2-7-migration-from-testcafe-v0-x-y-to-v1-0-0.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title: Migration from TestCafe v0.x.y to v1.0.0
+permalink: /blog/:title.html
+---
+# Migration from TestCafe v0.x.y to v1.0.0
+
+TestCafe v1.0.0 release introduces minor changes to the behavior and programming interface. This document lists these changes and describes how to address them.
```

Comment:
TestCafe v1.0.0 release -> TestCafe v1.0.0

Reply:
"to the behavior" The behavior of what?

Reply:
"describes how to address them" makes it sound like you created some issues/problems.

Reply:
a) *behavior*: I thought that if we speak about TestCafe, we mean **its** behavior, but, okay, we can say:

*TestCafe v1.0.0 introduces minor changes to the framework's behavior and programming interface.*

or

*Version 1.0.0 introduces minor changes to TestCafe behavior and programming interface.*

b) *address*: I didn't use the expression "breaking change" here, but I did mean breaking changes. So, yes, they are sort of issues for our customers. Can breaking changes be "addressed" or is it still incorrect? We can say something more neutral instead, like:

*This document lists these changes and describes how to migrate to the new version.*